CONTRIBUTING.md

```diff
@@ -0,0 +1,71 @@
+# Contributing
+
+Thank you for wanting to contribute to *aws-nuke*.
+
+Because of the amount of AWS services and their rate of change, we rely on your
+participation. For the same reason we can only act retroactive on changes of
+AWS services. Otherwise it would be a fulltime job to keep up with AWS.
+
+
+## How Can I Contribute?
+
+### Some Resource Is Not Supported by *aws-nuke*
+
+If a resource is not supported yet by *aws-nuke*, you have two options to
+resolve this:
+
+* File [an issue](https://github.com/rebuy-de/aws-nuke/issues/new) and describe
+  what resource is missing. We will take care and add it as soon as we have
+  some time.
+* Add the resource yourself and open a Pull-Request. Please follow the
+  guidelines below to see how to create such a resource.
+
+
+### Some Resource Does Not Get Deleted
+
+Please check the following points before creating a bug issue:
+
+* Is the resource actually supported by *aws-nuke*? If not, please follow the
+  guidelines above.
+* Are there permission problems? In this case *aws-nuke* will print errors,
+  that usually contain the status code `403`.
+* Did you just get scared by an error that was printed? *aws-nuke* does not
+  know about dependencies between resources. To work around this it will just
+  retry deleting all resources in multiple iterations. Therefore it is normal
+  that there are a lot of dependency errors in the first one. The iterations
+  are separated by lines starting with `Removal requested: ` and only the
+  errors in the last block indicate actual errros.
+
+File [an issue](https://github.com/rebuy-de/aws-nuke/issues/new) and describe
+as accurately as possible how to generate the resource on AWS that cause the
+errors in *aws-nuke*. Ideally this is provided in a reproducable way like
+a Terraform template or AWS CLI commands.
+
+
+### I Have Ideas to Improve *aws-nuke*
+
+### I Just Have a Question
+
+Join our ICR channel [`#aws-nuke`](https://webchat.freenode.net/?channels=aws-nuke) at `cat.freenode.net`.
```

Comment:
I somehow would prefer some kind of mailing list here, because I am not sure if we can give a proper coverage in a IRC channel.